src/components/media-views.js

```diff
@@ -319,15 +327,17 @@ AFRAME.registerComponent("media-video", {
         return;
       }
 
-      texture.audioSource = this.el.sceneEl.audioListener.context.createMediaElementSource(texture.image);
-      this.video = texture.image;
+      if (!src.startsWith("webrtc://")) {
```

Comment:
I know our video streams happen to not have audio, but why special case this?

Reply:
i'll verify but this was breaking if i didn't avoid it iirc

Reply:
yes get this error in firefox `The captured HTMLMediaElement is playing a MediaStream. Applying volume or mute status is not currently supported.`

Reply:
will leave a comment for now